CHANGELOG.md

```diff
@@ -0,0 +1,160 @@
+- New Analyzer: MicrosoftKeyAnalyzer (8927204d901ed4275e2995239d2be66fc3e74925)
+- New Analyzer: AWSAccessKeyAnalyzer (ebc6eab54a11fd7ad9dd1959cdc29f314efd9d82)
+- New Analyzer: AWSSecretKeyAnalyzer (d07021a93ac0f64a86393aa3955771643b8e23e5)
+- New Analyzer: SlackWebhookAnalyzer (c40c3647489e8fc6e5e3b0c3e3f9dc5a1d2fd5cb)
+- New Analyzer: GoogleOAuthKeyAnalyzer (fbfb8bf30e1acab0375b3d388cf92e13af696fd5)
+- New Analyzer: FacebookAccessTokenAnalyzer (bb51e3e82d954a4a75f478c675e7a7f2c2b9761d)
+- New Analyzer: Base64Analyzer (8d50fbe23a31e7e9e4418c4d95dd281c863b9887)
+- New Analyzer: AdobeKeyAnalyzer (4e52345b63ded044ea13744901c728fec7f57aab)
+- New Analyzer: EmailPasswordPairAnalyzer (f0af9cbe660c322db7de31676a92186b49f6fa36)
+- New Analyzer: HashAnalyzer (87080c2c3fe1b7ff9f5d200b484061f3b9e6c8e0)
+- New Analyzer: SlackTokenAnalyzer (d68616917fd36f33030741a71c0037dc3274d9f4)
+- New Analyzer: MailChimpApiKeyAnalyzer (2e5302d6faed2acf1e626950b6f0d91fa88e2a09)
+- New Analyzer: MegaLinkAnalyzer (c884cb60dd17c191aba637dcb8dc4d6675dfc101)
+- New Analyzer: StripeApiKeyAnalyzer (f9bd202d0813aebb6bc3f189a43158227ca2bdea)
+- New Analyzer: AzureSubscriptionKeyAnalyzer (b010cb58979c009be943e9f5dd36432e834bda41)
+- New Analyzer: GoogleApiKeyAnalyzer (635a5e4f1357e458d2d35c7dbd8732d756694ec1)
+
+### Changed
+- Add pastebinscraper by default (d00fc83dbf094e4fadedb4c5d9b22ad6e8733ace)
+- Remove unused custom_payload from DiscordAction (7b13d757242af33ddc9ce1c2c3e5e3d7618cd4b5)
+
+### Fixed
+- SHA hash analyzer can now accept multiple length hashes (494d1af5871629bbfd2ee6c859e360e90042ae18)
+- Use empty string if paste.body is set to None in URL- and IBANAnalyzer (09f6763892f22dc6488abfd9d03f73eab232e4ca)
+- Include some changes when creating a sqlite file (0eb3504e9ad67cea4cd808fd7edb7f4ec862264d)
+
+## [1.0.16] - 2019-09-08
+### Added
+- Perform checks on pastebin responses to catch errors (01f865e4acf4f7a92302a28010498732b3514d85)
+- If pastes are not ready for downloading, requeue them (01f865e4acf4f7a92302a28010498732b3514d85)
+
+## [1.0.15] - 2019-09-04
+### Added
+- Ability to search for multiple words in single WordAnalyzer (d2a7e0957ec1c758be2e76441e50f12b7a08a575)
+- Ability to restart running scrapers after adding a new one (de99892f56e0933fee2939bac83c91fad3782994)
+- Ability to register error handlers (1fae47e0b97f5d42966775d305a6b5561db1dc73)
+### Fixed
+- Check if paste is None before analyzing it (2fd7b398ab13c3d891b4f0164625ce38f25f205f, f4bfa466c27329839e7155b7f625ef18575ab1a8)
+- Broken behaviour for WordAnalyzer blacklist (df2dd5b1627915f1b3996e7c4a10349ad837f7e2)
+- Reduced sleep time in order to shut down pastepwn faster (55bb18d17ee27e443b244dcd3ea2a0362fc5853a)
+- Add check in GenericAnalyzer if parameter is callable (781d6d0324fb73d93f9d8789aa08a0e72a597825)
+- WordAnalyzer not matching in case sensitive mode (8762dddc2a41f01db45549006d8a9707fa1203a6)
+
+## [1.0.14] - 2019-09-04
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/manual/d-Rickyy-b/pastepwn/pullRequest?prid=4945865)